release_notes/371-release-notes.adoc

```diff
@@ -126,6 +126,10 @@ The default value for the `--include-snoozed` option of the `roxctl image scan`
 
 You can create diagnostic bundles to assist Red Hat in supporting you with {product-title-short}. This diagnostic bundle has been enhanced to include notifiers, auth providers and auth provider groups, access control roles with attached permission set and access scope, and system configuration information. Users with the `DebugLogs` permission can read listed entities from a generated diagnostic bundle regardless of their respective permissions. 
 
+[id="ocp4-cis-benchmarks"]
+=== Align OCP4-CIS scanning benchmarks control numbers
+
+We added the CIS control number to compliance scan results to enable customers to reference the original control from the CIS benchmark standard *(ROX-9986)*.
```

Comment:
> ! we pronoun
Do not use ... as in "we recommend". Do not use to refer collectively to both the authors and the audience, as in "we can proceed to the next step". Use only in the question portion of frequently asked questions (FAQs), in marketing proposals, or in information that has listed authors and in which the authors describe their own actions or opinions.

Reply:
fixed! TY